packages/runtime/container-runtime/src/test/dataStoreContext.spec.ts

```diff
@@ -287,7 +291,7 @@ describe("Data Store Context Tests", () => {
 
     describe("RemoteDataStoreContext", () => {
         let remotedDataStoreContext: RemotedFluidDataStoreContext;
-        let dataStoreAttributes: IFluidDataStoreAttributes;
+        let dataStoreAttributes: ReadFluidDataStoreAttributes;
```

Comment:
Can we add / updates these tests to include `disableIsolatedChannels` attributes.